packages/amplify_authenticator/lib/src/widgets/form.dart

```diff
@@ -153,11 +147,11 @@ class SignUpForm extends AuthenticatorForm<SignUpForm> {
         );
 
   /// Whether to include the attributes specified by your Amplify configuration.
-  final bool includeDefaultAttributes;
+  final bool useAttributesFromConfig;
 
   @override
-  List<AuthenticatorFormField> additionalFields(BuildContext context) {
-    if (!includeDefaultAttributes) {
+  List<SignUpFormField> runtimeFields(BuildContext context) {
+    if (!useAttributesFromConfig || fields.isNotEmpty) {
```

Comment:
If we're not gonna allow both custom fields and auto-attribute fields, then we should remove the option from the constructor, since it doesn't matter.

Reply:
We could actually include both... the tricky thing would be visually ordering the fields in a way that makes sense. I'll give it a try, and we could refine as needed. Potentially some sort of sort order value on the fields?

Reply:
Hmm... yeah, I'm going to remove the option for now until we figure out a good API for creating a union of custom and config values. I think an experience where you pass a custom form, and all you see are the values you pass, is fairly simple and makes sense.